Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.8.13-slim-buster
+
+RUN mkdir -p /app
+COPY . main.py /app/
+WORKDIR /app
+RUN pip install -r requirements.txt
+RUN python -m textblob.download_corpora
+EXPOSE 8080
+CMD [ "main.py" ]
+ENTRYPOINT [ "python" ]
```

Comment:
Is there a way we can use just one of these? Also... This might not work as we rely on running uvicorn to get the app running. Take a look at the README.md file to see the commands needed to run the app.

Reply:
Alright